### PR TITLE
[Maintenance] Remove deprecated parameters and change factory name

### PR DESCRIPTION
**Pull request by Wojdylak**
| Q               | A
| --------------- | -----
| Branch?         | 2.0
| Bug fix?        | no
| New feature?    | no
| Related tickets | #315

### CODE REVIEW

File: tests/Behat/Context/Admin/ManagingPaymentMethodsContext.php [hunk 1]
```diff
@@ -52,7 +52,7 @@ public function yesterdayReportCsvFileShouldBeSuccessfullyDownloaded(): void
      */
     public function iTryToCreateANewPaymentMethodWithGatewayFactory(): void
     {
-        $this->createPage->tryToOpen(['factory' => 'sylius.pay_pal']);
+        $this->createPage->tryToOpen(['factory' => 'sylius_paypal']);
```
Comment (NoResponseMate) [inline tests/Behat/Context/Admin/ManagingPaymentMethodsContext.php]:
```suggestion
        $this->createPage->tryToOpen(['factory' => SyliusPayPalExtension::PAYPAL_FACTORY_NAME]);
```

File: templates/shop/checkout/select_payment/content/form/select_payment/payment/choice/details/paypal.html.twig [hunk 2]
```diff
@@ -1,4 +1,4 @@
-{% if hookable_metadata.context.method.gatewayConfig.factoryName == 'sylius.pay_pal' %}
+{% if hookable_metadata.context.method.gatewayConfig.factoryName == constant('Sylius\\PayPalPlugin\\DependencyInjection\\SyliusPayPalExtension::PAYPAL_FACTORY_NAME') %}
```
Comment (NoResponseMate) [inline templates/shop/checkout/select_payment/content/form/select_payment/payment/choice/details/paypal.html.twig]:
```suggestion
{% if hookable_metadata.context.method.gatewayConfig.factoryName|default('') == constant('Sylius\\PayPalPlugin\\DependencyInjection\\SyliusPayPalExtension::PAYPAL_FACTORY_NAME') %}
```

File: templates/admin/payment_method/form/header/paypal_logo.html.twig [hunk 3]
```diff
@@ -1,4 +1,4 @@
-{% if hookable_metadata.context.resource.gatewayConfig.factoryName == 'sylius.pay_pal' %}
+{% if hookable_metadata.context.resource.gatewayConfig.factoryName == constant('Sylius\\PayPalPlugin\\DependencyInjection\\SyliusPayPalExtension::PAYPAL_FACTORY_NAME') %}
```
Comment (NoResponseMate) [inline templates/admin/payment_method/form/header/paypal_logo.html.twig]:
```suggestion
{% if hookable_metadata.context.resource.gatewayConfig.factoryName|default('') == constant('Sylius\\PayPalPlugin\\DependencyInjection\\SyliusPayPalExtension::PAYPAL_FACTORY_NAME') %}
```

File: spec/Processor/PayPalPaymentRefundProcessorSpec.php [hunk 4]
```diff
@@ -104,15 +104,15 @@ function it_does_nothing_if_payment_is_not_pay_pal(
         $this->refund($payment);
     }
 
-    function it_does_nothing_if_payment_is_payment_has_not_pay_pal_order_id(
+    function it_does_nothing_if_payment_is_payment_has_not_paypal_order_id(
```
Comment (TheMilek) [inline spec/Processor/PayPalPaymentRefundProcessorSpec.php:107]:
```suggestion
    function it_does_nothing_if_payment_is_payment_that_has_not_paypal_order_id(
```

File: UPGRADE-2.0.md [hunk 5]
```diff
@@ -147,6 +155,19 @@
...
+   | Old parameter                            | New parameter                            | 
+   |------------------------------------------|------------------------------------------|
+   | `sylius.paypal.prioritized_factory_name` | `sylius_paypal.prioritized_factory_name` |
+   | `sylius.pay_pal.request_trials_limit`    | `sylius_paypal.request_trials_limit`     |
+   | `sylius.paypal.logging.increased`        | `sylius_paypal.logging.increased`        |
+   | `sylius.pay_pal.facilitator_url`         | `sylius_paypal.facilitator_url`          |
+   | `sylius.pay_pal.api_base_url`            | `sylius_paypal.api_base_url`             |
+   | `sylius.pay_pal.reports_sftp_host`       | `sylius_paypal.reports_sftp_host`        |
+
+1. The parameter `sylius.paypal.logging_level_increased` has been removed.
```
Comment (TheMilek) [inline UPGRADE-2.0.md:169]: Why? Was it duplicated by `sylius_paypal.logging.increased`?

**Review by TheMilek — APPROVED** [review]

**Review by GSadee — APPROVED** [review]

**Pull request merged**